system/ee/ExpressionEngine/View/addons/index.php

```diff
@@ -1,348 +1,290 @@
-        			</div>
-          	</div>
-          </div>
-        </div>
-      </div>
-      <div class="table-responsive table-responsive--collapsible">
-        <table cellspacing="0">
-  				<thead>
-      			<tr class="app-listing__row app-listing__row--head">
-							<th class="column-sort-header column-sort-header--active">
-								<a href="" class="column-sort column-sort--desc">Name</a>
+	<div class="tbl-ctrls">
+		<div class="panel-heading">
+			<div class="app-notice-wrap"><?= ee('CP/Alert')->getAllInlines() ?></div>
+			<?php if (!empty($updates) && count($updates) == 1) : ?>
+				<div id="alert_banner" class="alert">
+					<div class="alert__icon"><i class="fas fa-info-circle fa-fw"></i></div>
+					<div id="alert" href="" class="alert__close">
+						<a id="alert_close" class="fas fa-times alert__close-icon"></a>
+					</div>
+					<p class="alert__title"> <br>
+					<p class="alert__title"><?php echo lang('update'); ?><br>
+						<?php echo lang('there_is'); ?> <strong><?= count($updates) ?></strong> <?php echo lang('single_update'); ?> <a id="ShowMe"><?php echo lang('show'); ?></a></p>
+				</div>
+			<?php endif; ?>
+
+			<?php if (!empty($updates) && count($updates) > 1) : ?>
+				<div id="alert_banner" class="alert">
+					<div class="alert__icon"><i class="fas fa-info-circle fa-fw"></i></div>
+					<div id="alert" href="" class="alert__close">
+						<a id="alert_close" class="fas fa-times alert__close-icon"></a>
+					</div>
+					<p class="alert__title"><?php echo lang('updates'); ?><br>
+
+						<?php echo lang('there_are'); ?> <strong><?= count($updates) ?></strong> <?php echo lang('multi_updates'); ?> <a id="ShowMe"><?php echo lang('show'); ?></a></p>
+					<p></p>
+				</div>
+			<?php endif; ?>
+
+
+			<div class="form-btns form-btns-top">
+				<div class="title-bar js-filters-collapsable title-bar--large">
+					<div class="filter-bar">
+						<div class="filter-bar__item ">
+							<div class="filter-search-bar__item ">
+								<button type="button" class="has-sub filter-bar__button js-dropdown-toggle button button--default button--small" data-filter-label="status" title="status">status</button>
+								<div class="dropdown">
+									<div class="dropdown__scroll">
+										<a class="dropdown__link" id="Installed_Sort"><?php echo lang('installed'); ?></a>
+										<a class="dropdown__link" id="Uninstalled_Sort"><?php echo lang('uninstalled'); ?></a>
+										<a class="dropdown__link" id="Update_Sort"><?php echo lang('update_available'); ?></a>
+									</div>
+								</div>
+							</div>
+						</div>
+						<div class="filter-bar__item filter-search-form">
+							<div class="filter-search-bar__item">
+								<div class="search-input">
+									<input id="Search_Term" class="search-input__input input--small" type="text" name="filter_by_keyword" value="" placeholder="Search">
+								</div>
+							</div>
+						</div>
+
+						<script src="themes\ee\cp\js\src\cp\add-ons-index.js">
```

Comment:
This is not how we include scripts. Please use the combo loader in the addons controller.
https://docs.expressionengine.com/latest/development/legacy/libraries/cp.html#add-javascript-files-to-the-javascript-combo-loader